app/src/test/java/com/elavon/converge/BaseTest.java

```diff
@@ -1,23 +1,34 @@
 package com.elavon.converge;
 
+import com.elavon.converge.xml.XmlMapper;
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
 
-import org.junit.Before;
+import org.junit.BeforeClass;
 
 public abstract class BaseTest {
 
-    private Gson gson;
+    private static XmlMapper xmlMapper;
+    private static Gson gson;
 
-    @Before
-    public void beforeClass() {
+    @BeforeClass
+    public static void beforeClass() {
+        xmlMapper = new XmlMapper();
         gson = new GsonBuilder().setPrettyPrinting().create();
     }
 
     protected void print(Object o) {
         System.out.println(o);
     }
 
+    protected void printXml(Object o) {
+        try {
+            System.out.println(xmlMapper.write(o));
```

Comment:
oh gotta move to a logger and not use System.out - not important now but may be in the next changes...

Reply:
I don't usually use logger in test classes. Did you mean in the main classes?

Reply:
yup - sorry yes that's what I meant.